x-pack/plugin/esql/qa/testFixtures/src/main/resources/semantic_text.csv-spec

```diff
@@ -173,3 +173,946 @@ host:keyword | semantic_text_field:semantic_text
+
+FROM semantic_text
+| EVAL result = repeat(semantic_text_field, 2)
+| WHERE length(semantic_text_field) < 25
+| KEEP result
+;
+
+result:keyword
+live long and prosperlive long and prosper
+;
+
+replace
+required_capability: semantic_text_type
+
+FROM semantic_text
+| EVAL result = replace(semantic_text_field, "excellent", "good")
+| WHERE length(semantic_text_field) < 30
+| KEEP result
+| SORT result
+;
+
+result:keyword
+be good to each other
+live long and prosper
+;
+
+right
+required_capability: semantic_text_type
+
+FROM semantic_text
+| EVAL result = right(semantic_text_field, 2)
+| KEEP result
+| SORT result
+;
+
+result:keyword
+er
+er
+us
+;
+
+rtrim
+required_capability: semantic_text_type
+
+FROM semantic_text
+| EVAL result = rtrim(semantic_text_field)
+| KEEP result
+| SORT result
+;
+
+result:semantic_text
+all we have to decide is what to do with the time that is given to us
+be excellent to each other
+live long and prosper
+;
+
+split
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = split(st_version, ".")
+| SORT _id
+| KEEP result
+;
+
+result:keyword
+["1", "2", "3"]
+["9", "0", "0"]
+null
+;
+
+startsWith
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = starts_with(semantic_text_field, "be")
+| KEEP _id, result
+| SORT _id
+;
+
+_id:keyword | result:bool
+1           | false
+2           | false
+3           | true
+;
+
+substring
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = substring(semantic_text_field, 2, 1)
+| KEEP _id, result
+| SORT _id
+;
+
+_id:keyword | result:keyword
+1           | i
+2           | l
+3           | e
+;
+
+toBase64
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = to_base64(st_integer)
+| KEEP _id, result
+| SORT _id
+;
+
+_id:keyword | result:keyword
+1           | MjM=
+2           | MTIy
+3           | null
+;
+
+toLower
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = to_lower(st_cartesian_point)
+| KEEP _id, result
+| SORT _id
+;
+
+_id:keyword | result:semantic_text
+1           | point(4297.11 -1475.53)
+2           | point(7580.93 2272.77)
+3           | null
+;
+
+toUpper
+required_capability: semantic_text_type
+
+FROM semantic_text METADATA _id
+| EVAL result = to_upper(semantic_text_field)
+| KEEP _id, result
+| SORT _id
+; 
+
+_id:keyword | result:semantic_text
+1           | LIVE LONG AND PROSPER
+2           | ALL WE HAVE TO DECIDE IS WHAT TO DO WITH THE TIME THAT IS GIVEN TO US
+3           | BE EXCELLENT TO EACH OTHER
+;
+
+trim
+required_capability: semantic_text_type
+
+FROM semantic_text
+| EVAL result = trim(semantic_text_field)
+| SORT result
+| KEEP result
+;
+
+result:semantic_text
+all we have to decide is what to do with the time that is given to us
+be excellent to each other
+live long and prosper
+;
+
+mvAppend
```

Comment:
tests for [Multi value functions](https://www.elastic.co/guide/en/elasticsearch/reference/master/esql-functions-operators.html#esql-mv-functions)